test/redirector-test.js

```diff
@@ -67,3 +80,47 @@ describe('Redirector Worker', function() {
     });
 });
 
+describe('Configuration Data', function() {
+    it('should be able to check data from redirector', async function() {
+        expect(global.experimentPages).to.be.an('array');
+    });
+
+    it('should have good content in experiment pages hashes', async function() {
+        global.experimentPages.forEach(function (value) {
+            expect(value).to.have.property('targetPath');
+            expect(value).to.have.property('sandboxPath');
+            expect(value).to.have.property('sampleRate');
+
+            // A common mistake might be to accidentally point at same source and destination
+            // This test ensures there is some difference.
+            expect(value['targetPath']).to.not.be.equal(value['sandboxPath']);
+
+            // Doing some light testing that the paths are reasonable
+            // when merged with a domain, they ought to be valid urls
+            // otherwise nothing will ever reach them.
+            expect(isValidPath(value['targetPath'])).to.be.true;
+            expect(isValidPath(value['sandboxPath'])).to.be.true;
+
+
+            expect(value['sampleRate']).to.be.above(0);
+            expect(value['sampleRate']).to.be.below(1);
+            // TODO: write tests for other attributes
```

Comment:
Noting TODO here? (not sure if this is something you wanted to tackle here, or just making a note for future updates 🙂)